### ISSUE TITLE
[generate:entity:content] Incorrect namespace and definition

### ISSUE THREAD

**Issue by mgrimard** [hechoendrupal/DrupalConsole#1495]
In 0.9.10 the entity created by generate:entity:content seems to have some of it's namespacing incorrect. In each of the entity forms, it declares it's namespace as ```namespace Drupal\my_module\Entity\Form;``` but it is located in Drupal\my_module\Form. Additionally, in the Entity class it also declares the forms as located in that same namespace.

``` 
# \Drupal\my_module\Entity\MyEntityItem
/**
 * Defines the KFPL PMC Item entity.
*------- snip --------
*     "form" = {
 *       "default" = "Drupal\my_module\Entity\Form\MyEntityItemForm",
 *       "add" = "Drupal\my_module\Entity\Form\MyEntityItemForm",
 *       "edit" = "Drupal\my_module\Entity\Form\MyEntityItemForm",
 *       "delete" = "Drupal\my_module\Entity\Form\MyEntityItemDeleteForm",
 *     },
```

**Comment by mnico** [comment]
This was solved with this PR #1532

**Comment by jmolivas** [comment]
Thanks @mnico
